jerry-core/ecma/builtin-objects/ecma-builtin-promise.c

```diff
@@ -359,21 +355,21 @@ ecma_builtin_promise_perform_all (ecma_value_t iterator, /**< iteratorRecord */
 exit:
   ecma_free_value (remaining);
   ecma_deref_object (values_array_obj_p);
-  ecma_deref_object (resolve_func_p);
 
   return ret_value;
-} /* ecma_builtin_promise_perform_all */
+} /* ecma_builtin_promise_perform_all_or_all_settled */
 
 /**
- * The common function for both Promise.race and Promise.all.
+ * The common function for Promise.race, Promise.all and Promise.allSettled.
  *
  * @return ecma value of the new promise.
  *         Returned value must be freed with ecma_free_value.
  */
 static ecma_value_t
-ecma_builtin_promise_race_or_all (ecma_value_t this_arg, /**< 'this' argument */
-                                  ecma_value_t iterable, /**< the items to be resolved */
-                                  bool is_race) /**< indicates whether it is race function */
+ecma_builtin_promise_helper (ecma_value_t this_arg, /**< 'this' argument */
+                             ecma_value_t iterable, /**< the items to be resolved */
+                             uint8_t builtin_routine_id) /**< built-in wide routine
+                                                          *   identifier */
```

Comment:
Do we need to break the comment into two lines? There was a similar case above as well.